### ISSUE TITLE
gdal_calc windows subprocess error

### ISSUE THREAD

**Issue by ogaking** [iamdonovan/spymicmac#52]
Error encountered running post_process_micmac whilst assigning CRS to DEM. 

Spymicmac version:
0.2.1.dev41+g09496fc

```
(spymicmac) C:\Users\nok25\spymicmac_processing\SantaCruz>post_process_micmac "EPSG:32718" DEM_UTM.tif MEC-Malt
C:\Users\nok25\Software\spymicmac\src\spymicmac\image.py:22: NumbaDeprecationWarning: The 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.
  def nanstd(a):
Input file size is 17423, 19452
0...10...20...30...40...50...60...70...80...90...100 - done.
Input file size is 17423, 19452
0...10...20...30...40...50...60...70...80...90...100 - done.
Input file size is 17423, 19452
0...10...20...30...40...50...60...70...80...90...100 - done.
Traceback (most recent call last):
  File "C:\Users\nok25\AppData\Local\anaconda3\envs\spymicmac\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\nok25\AppData\Local\anaconda3\envs\spymicmac\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\nok25\AppData\Local\anaconda3\envs\spymicmac\Scripts\post_process_micmac.exe\__main__.py", line 7, in <module>
  File "C:\Users\nok25\Software\spymicmac\src\spymicmac\tools\post_process_micmac.py", line 27, in main
    post_process(**vars(args))
  File "C:\Users\nok25\Software\spymicmac\src\spymicmac\micmac.py", line 1225, in post_process
    subprocess.Popen(['gdal_calc.py', '--quiet', '-A', 'tmp_mask.tif', '-B', 'tmp_geo.tif',
  File "C:\Users\nok25\AppData\Local\anaconda3\envs\spymicmac\lib\subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\nok25\AppData\Local\anaconda3\envs\spymicmac\lib\subprocess.py", line 1456, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
OSError: [WinError 193] %1 is not a valid Win32 application

```

**Comment by iamdonovan** [comment]
Please try updating with the commit on `winfix2` (https://github.com/iamdonovan/spymicmac/commit/554f6db09ce52c05942804af8b35c94fceeca04b), then re-run `post_process_micmac`.

**Comment by iamdonovan** [comment]
This should be fixed by #55.
